### DISCUSSION TITLE
Tablo filtremelerine UX açısından loading eklenmesi

### DISCUSSION THREAD

**Discussion by eece** [acikkaynak/deprem-yardim-projesi#56] [Feedback]
Tablo olan bir ekranda sorgu yapılırken sorgu sonucu gelesiye kadar tabloya bir loading eklenebilir. Böylece sonuç bekleniyor mu, boş mu geldi anlaşılmış olur. Şu anda butona basıldığında anlık tepki vermediği için sayfayı kapatıp gidenler oluyor.  Örnek sayfa:
![image](https://user-images.githubusercontent.com/18285160/218222644-f3741fb3-5bc5-4a5a-8d96-de10ec5a2716.png)

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz ilgili ekibe iletildi.
